src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    /**
+     * In your implementations - create 2-3 models, add them to the list and return
+     *
+     * @return - the list of models
+     */
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            bulldozers.add(new Bulldozer());
+        }

```

Comment:
I think we can make number 3 as constant field, and lets set some values to Buldozzers fields (some `tracked` and `volume`)

same for other MachineProdusers